app/main.py

```diff
@@ -1,3 +1,48 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float | Distance) -> Distance:
+        if type(other) == Distance:
+            return Distance(self.km + other.km)
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: int | float | Distance) -> Distance:
+        if type(other) == Distance:
+            self.km += other.km
+        else:
+            self.km += other
+        return self
+
+    def __mul__(self, other) -> Distance:
+        return Distance(self.km * other)
+
+    def __truediv__(self, other) -> Distance:
+        return Distance(round((self.km / other), 2))
+
+    def __lt__(self, other) -> bool:
+        return self.km < other
+
+    def __gt__(self, other) -> bool:
+        return self.km > other
+
+    def __eq__(self, other) -> bool:
+        return self.km == other
+
+    def __le__(self, other) -> bool:
+        return self.km <= other
+
+    def __ge__(self, other) -> bool:
+        return self.km >= other
```

Comment:
try to use the previously created magic method in these two methods to simplify your code
Hint: `<=` is the same as `not >`